umbraco/Umbraco-CMS
# Packages Modal - Further Fixes for Modal Trap

### Prerequisites


Related to issue: [Fix A & AA accessibility issues throughout Backoffice](https://github.com/umbraco/Umbraco-CMS/issues/12546)

If there's an existing issue for this PR then this fixes:
- [umbraco/Umbraco-CMS.Accessibility.Issues#61](https://github.com/umbraco/Umbraco-CMS.Accessibility.Issues/issues/61) (Package Details - Image Modal - Missing Modal Trap)

### Description
**What did you do and why ?**
- Updated the focus trap when the modal is open so the user is only able to navigate to the elements of the modal (and not outside onto the browser)

**How to test ?**
- Go to the Packages page, and use the tab key to navigate through the page to the images, when on an image click enter and open up the Image Modal
- Once open use the TAB key to navigate through the elements in the modal check that the focus stays in the modal and doesn't allow the user to navigate the package page.
